src/deprecation.js

```diff
@@ -601,6 +601,14 @@ export default function deprecation(core)
         return this.generateCanvasTexture(scaleMode, resolution);
     };
 
+    /**
+     * @method
+     * @name PIXI.GroupD8.isSwapWidthHeight
+     * @see PIXI.GroupD8.isVertical
+     * @param {number} rotation - The number to check.
```

Comment:
On deprecated methods we typically remove the params and return values so people won't use it.

Reply:
Lint forces me to do that.

Reply:
also, no, you dont. Look further after my method there are many others

Reply:
Actually, that's not important ;) I suspect that the only ones who are using this function are I and @finscn :)